domain/user/service/service_test.go

```diff
@@ -92,63 +94,74 @@ func (s *serviceSuite) setMockState(c *gc.C) map[string]stateUser {
 		gomock.Any(), gomock.Any(),
```

Comment:
comment for above. It looks like we are keying mockState map on UUID now.

Line 91 needs to change to `mockState := map[user.UUID]stateUser{}`

Reply:
not sure that it will be suitable for all our tests.

Reply:
we're still using name as a life-hack for `AddUser*` func, because it's hard to get UUID that should be generated.

Reply:
done